content/docs/2.6/scalers/azure-app-insights.md

```diff
@@ -0,0 +1,117 @@
++++
+title = "Azure Application Insights"
+layout = "scaler"
+availability = "v2.6+"
+maintainer = "Community"
+description = "Scale applications based on Azure Application Insights metrics."
+go_file = "azure_app_insights_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-app-insights` trigger that scales based on an Azure Application Insights metric.
+
+```yaml
+triggers:
+- type: azure-app-insights
+  metadata:
+    metricAggregationTimespan: "0:1"
+    metricAggregationType: avg
+    metricFilter: cloud/roleName eq 'role_name'
+    metricName: "customMetrics/example-metric"
```

Comment:
Is this a `metricName` for KEDA internals? If so, we shouldn't requrie users to specify this. We should generate one, take a look on the last few scalers how they done this.

Reply:
This is the metric to retrieve from app insights. Some example are availabilityResults/availabilityPercentage, browserTimings/networkDuration, and customMetrics/customMetricForSomeService). The REST API described [here](https://dev.applicationinsights.io/reference) uses the term metricId. Maybe that's better?

Reply:
Gotcha, in this case I think that `metricId` is better, so we are aligned with the service and avoid confusion with KEDA's `metricName`

Reply:
Sounds good. I'll make that change later today.

Reply:
Sounds good to me!

Reply:
metricName has been renamed to metricId